superset/common/query_actions.py

```diff
@@ -154,6 +153,7 @@ def _get_samples(
             qry_obj_cols.append(o.get("column_name"))
         else:
             qry_obj_cols.append(o.column_name)
+    query_obj.orderby = [(query_obj.columns[0], True)]
```

Comment:
I don't have much context here but given that we're explicitly setting:

```
query_obj.orderby = []
query_obj.metrics = None
```
and that the definition of `OrderBy` is:

```
OrderBy = tuple[Metric, bool]
```

Setting the order by to the first column might not be the correct answer given that it only accepts ordering by metric which is explicitly being set to empty. Maybe @zhaoyongjie or @villebro have more context and can help us.

Reply:
Thanks for the feedback @michael-s-molina 🙏 !  Someone in the associated issue thread said this one-liner was a solution for them so I hoped it would be this simple but it looks like not.

@zhaoyongjie gave a [high-level description of how he thinks this should be fixed](https://github.com/apache/superset/issues/24072#issuecomment-1859252019).  Implementing it is outside of my current understanding unfortunately.

Reply:
Evan suggested changing the definition of OrderBy and that seems to have worked.